Python/optimizer_analysis.c

```diff
@@ -696,6 +703,22 @@ remove_unneeded_uops(_PyUOpInstruction *buffer, int buffer_size)
                 }
                 last_set_ip = pc;
                 break;
+            case _POP_TOP:
```

Comment:
We need to unify the zapping of useless instructions along with constants. They are logically the same. We can do that in another PR.

Reply:
Probably best done in another pass, but we can worry about that later.